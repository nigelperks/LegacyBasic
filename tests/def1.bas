10 REM DEF PARAMETER SHOULD NOT CHANGE VALUE OF VARIABLE OF THAT NAME
20 DEF F(X)=X*3
30 REM X UNDEDFINED
40 Y=F(9)
50 PRINT X,Y
60 REM X DEFINED
70 X=13
80 Y=F(4)
90 PRINT X,Y
100 END
