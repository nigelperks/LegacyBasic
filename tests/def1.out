 0       27 
 13      12 
